packages/components/src/components/modal/_modal.scss

```
@@ -84,6 +85,7 @@
     transition: transform $duration--moderate-02 motion(exit, expressive);
 
     @include carbon--breakpoint(md) {
+      position: initial;
```

```suggestion
      position: static;
```

oh I meant the actual initial value rather than `position: initial` since it's unsupported in IE